packages/node_modules/@node-red/nodes/core/sequence/17-split.js

```diff
@@ -314,11 +314,13 @@ module.exports = function(RED) {
                         if (err) {
                             return done(err);
                         }
-                        msgInfo.send({payload: result});
+                        msgInfo.msg.payload = result;
+                        msgInfo.send(msgInfo.msg);
```

Comment:
If we're now reusing `msgInfo.msg` should this get cloned before being sent?

Reply:
As there is one input and one output in reduce mode, is cloning necessary?

Reply:
I think you are right. As the reduce only happens when the "join" is complete then nothing should be able to change in that group of data after that point.